docs/README.md

```diff
@@ -1,7 +1,36 @@
-# nio Documentation
+# The nio Platform
 
 Welcome to nio! This documentation will help you design and build distributed systems with the nio Platform.
 
 nio is a platform for users like you to build real-world, innovative, and impactful solutions.
 
-niolabs welcomes your feedback on our documentation. If you have questions, comments, or suggestions, please email [editor@n.io](mailto:editor@n.io).
+Every state, every change, and every interaction in our world is a signal—from the percentage of moisture in the soil beneath your feet, to the vibration of your car motor, to the individual characters within your last tweet. Every _thing_ is producing context that already has, or will soon, become digitally available in the Internet of Things era. But how can these trillions of disparate signals work harmoniously to produce any meaningful value?
```

Comment:
This text was all previously on the "Getting Started" page but really reads like an "Introduction" anyway.